docs/syntax/origamijson.md

```diff
@@ -96,37 +96,29 @@ All origami components, whether modules or web services, should be discoverable
 	<td>string</td>
 	<td>(optional) For web services only, the URL on which the service is provided.  Required for web services.</td>
 </tr><tr>
+	<td><code>&nbsp;&nbsp;demosDefaults:&nbsp;{</code></td>
+	<td>object</td>
+	<td>(optional) Default options to be applied to all demos. Please check out the different options in the [modules component spec](http://origami.ft.com/docs/component-spec/modules/#demo-config)</td>
```

Comment:
Shouldn't we do this the other way around, ie link from the component spec to this doc?